polkadot/node/core/pvf/prepare-worker/src/lib.rs

```diff
@@ -35,22 +36,17 @@ use nix::{
 	unistd::{ForkResult, Pid},
 };
 use polkadot_node_core_pvf_common::{
-	executor_interface::{prepare, prevalidate},
```

Comment:
It looks messy, but I just merged two imports of `polkadot_node_core_pvf_common`. In fact, only `compute_checksum` is newly imported.